acoustid/future/fpindex/client.py

```
@@ -0,0 +1,284 @@
+
+                if (
+                    response.content_length
+                    and response.content_type == "application/json"
+                ):
+                    result = await response.json()
+                else:
+                    result = await response.text()
+
+                return status, result
+        except aiohttp.ClientError as e:
+            logger.error("Error making request: %s", e)
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)